projects/ngrx.io/content/guide/store/testing.md

```diff
@@ -59,7 +61,126 @@ describe('Auth Guard', () => {
 });
 </code-example>
 
-### Using Store for Integration Testing
+### Example: Effect injecting state
```

Comment:
This should be in `projects/ngrx.io/content/guide/effects/testing.md`

Reply:
Moved in 18e4a12.